packages/ui/v2/core/layouts/BookingLayout.tsx

```diff
@@ -11,10 +11,16 @@ const tabs: VerticalTabItemProps[] = [
     href: "/bookings/upcoming",
     icon: Icon.FiCalendar,
   },
+  // TODO: Add filter for unconfimred bookings in a future PR - Out of scope
```

Comment:
This filter doesn't work on the backend so was throwing errors - Will create a issue